src/main/resources/db/migration/V19032021_3__CPO-3-CasePaymentOrders.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE public.case_payment_orders (
```

Comment:
Does this file still need renaming?

Reply:
Yes, doing it now!

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/16/commits/caada6f64dfaa5a02352eb768ee9792564c98ff1